Example/Example/ServiceLocator.swift

```diff
@@ -42,18 +42,13 @@ class ServiceLocator {
 
     func provideCharactersNavigationController() -> CharactersNavigationController {
         let viewController = provideCharactersViewController()
-        return CharactersNavigationController(rootViewController: viewController)
+        let navigationController = CharactersNavigationController(rootViewController: viewController)
```

Comment:
I don't remember why I changed this code, I've reverted that change. The final code is:

```swift
 func provideCharactersNavigationController() -> CharactersNavigationController {
        let viewController = provideCharactersViewController()
        return CharactersNavigationController(rootViewController: viewController)
}
```